app/src/main/res/values/strings.xml

```diff
@@ -39,7 +39,9 @@
 
     <!-- Home - Recently saved bookmarks -->
     <!-- Title for the home screen section with recently saved bookmarks. -->
-    <string name="recent_bookmarks_title">Recent bookmarks</string>
+    <string name="recent_bookmarks_title" moz:RemovedIn="106" tools:ignore="UnusedResources">Recent bookmarks</string>
+    <!-- Title for the home screen section with recently saved bookmarks. -->
+    <string name="recent_bookmarks_title_2">Recently saved</string>
```

Comment:
```suggestion
    <string name="recently_saved_title">Recently saved</string>
```
Probably no need to keep the original name since the title is something else completely.